app/models/main.py

```diff
@@ -1266,6 +1274,7 @@ def copy(self):
             supplier=self.supplier,
             data=data,
             status='not-submitted',
+            lot_one_service_limit=self.lot.one_service_limit,
```

Comment:
This covers both copying from a live Service, and copying from another DraftService, correct?

Reply:
I think so, yes.

Reply:
Well, it's `from_service` that does the former.